pkg/systemd/journal.css

```diff
@@ -131,3 +131,8 @@
     white-space: nowrap;
     vertical-align: middle;
 }
+
+/* Make sure to not break log message lines in order to preserve information */
+#journal-entry .cockpit-info-table td {
+    white-space: normal;
```

Comment:
This doesn't stop a very long word from still sticking out to the right.  Is that OK?